pom.xml

```diff
@@ -30,8 +30,35 @@
 
     <properties>
         <shadeBase>org.weakref.jmx.\$internal</shadeBase>
+        <dep.plugin.njord.version>0.8.7</dep.plugin.njord.version>
+        <njord.publishingType>automatic</njord.publishingType>
+        <njord.waitForStates>true</njord.waitForStates>
+        <!-- Stage files to the local directory, njord will push them to central -->
+        <altDeploymentRepository>id::default::njord:</altDeploymentRepository>
+        <njord.publisher>sonatype-cp</njord.publisher>
+        <njord.releaseUrl>njord:template:release-sca</njord.releaseUrl>
+
+        <repository.snapshot.id>sonatype-central-portal</repository.snapshot.id>
+        <repository.snapshot.name>Sonatype Central Portal</repository.snapshot.name>
+        <repository.snapshot.url>https://central.sonatype.com/repository/maven-snapshots/</repository.snapshot.url>
+        <repository.release.id>sonatype-central-portal</repository.release.id>
+        <repository.release.name>Sonatype Central Portal</repository.release.name>
+        <repository.release.url>https://central.sonatype.com/repository/maven/</repository.release.url>
     </properties>
 
+    <distributionManagement>
+        <repository>
+            <id>${repository.release.id}</id>
+            <name>${repository.release.name}</name>
+            <url>${repository.release.url}</url>
+        </repository>
+        <snapshotRepository>
+            <id>${repository.snapshot.id}</id>
+            <name>${repository.snapshot.name}</name>
+            <url>${repository.snapshot.url}</url>
```

Comment:
These don't need to be parameterized. Inline them.

Reply:
Inlined